pkg/core/native/native_neo.go

```diff
@@ -192,18 +192,27 @@ func newNEO(cfg config.ProtocolConfiguration) *NEO {
 
 	desc = newDescriptor("registerCandidate", smartcontract.BoolType,
 		manifest.NewParameter("pubkey", smartcontract.PublicKeyType))
-	md = newMethodAndPrice(n.registerCandidate, 0, callflag.States)
+	md = newMethodAndPrice(n.registerCandidate, 0, callflag.States, config.HFDefault, config.HFEchidna)
+	n.AddMethod(md, desc)
+
+	md = newMethodAndPrice(n.registerCandidate, 0, callflag.States|callflag.AllowNotify, config.HFEchidna)
```

Comment:
BTW, each extension should be properly documented in `Hardforks` section of `./docs/node-configuration.md`.